psse/psse-model/src/main/java/com/powsybl/psse/model/io/Util.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.psse.model.io;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class Util {
+
+    private Util() {
+    }
+
+    public static String[] intersection(String[] strings1, String[] strings2) {
+        return ArrayUtils.removeElements(strings1, ArrayUtils.removeElements(strings1, strings2));
```

Comment:
Could maybe lead to a performance issue: I would implement this using sets, but not sure if it's interesting here

Reply:
The arrays used here are quite small, and the method is called only once per record group during write. But I will profile it using a big input file.

Reply:
We have to keep in mind that order of elements in first array must be preserved. It is not really a set intersection, maybe the name of the method should be changed to `keep(strings, stringsToKeep)`. 

Agree that a possible optimisation would be to build  a `Set` from `stringsToKeep` and iterate over `strings`, keeping each item only if it is found in the `stringsToKeep` set.